server/src/main/java/org/opensearch/action/search/SearchQueryCounters.java

```diff
@@ -48,16 +64,81 @@ public SearchQueryCounters(MetricsRegistry metricsRegistry) {
             "Counter for the number of top level and nested bool search queries",
             UNIT
         );
+        this.boostCounter = metricsRegistry.createCounter(
+            "search.query.type.boast.count",
+            "Counter for the number of top level and nested boast search queries",
+            UNIT
+        );
+        this.constantScoreCounter = metricsRegistry.createCounter(
+            "search.query.type.counstantscore.count",
```

Comment:
Are we using camel case here? If yes, this and few other places should be changed. If not, others like `disMax` should be changed